libraries/consul_watch.rb

```diff
@@ -11,12 +11,13 @@ module Resource
     # @since 1.0.0
     class ConsulWatch < Chef::Resource
       include Poise(fused: true)
+      include ConsulCookbook::Helpers
       provides(:consul_watch)
       default_action(:create)
 
       # @!attribute path
       # @return [String]
-      attribute(:path, kind_of: String, default: lazy { "/etc/consul/#{name}.json" })
+      attribute(:path, kind_of: String, default: lazy { join_path node['consul']['service']['config_dir'], "#{name}.json" })
```

Comment:
I know that I am going to sound overly pedantic here, but I prefer to hardcode the defaults and keep the node object out completely. Is there any functional difference if you were to do that?